source/frontend/src/features/mapSideBar/acquisition/AcquisitionContainer.tsx

```diff
@@ -275,7 +275,12 @@ export const AcquisitionContainer: React.FunctionComponent<IAcquisitionContainer
   };
 
   // UI components
-  if (loadingAcquisitionFile || (loadingAcquisitionFileProperties && !isPropertySelector)) {
+  if (
+    loadingAcquisitionFile ||
+    (loadingAcquisitionFileProperties && !isPropertySelector) ||
+    file?.fileType !== FileTypes.Acquisition ||
```

Comment:
this is odd. Do you know when this would be true?

Reply:
Yes, so essentially there is a race condition where when the file is loading the previous file is still in context. The state transition to set loading to false occurs before the new file is set into the context, and so this occurs.

Another solution would be to set the file in the context to null immediately during load.

Reply:
This bug happens when you load a Research file into the sidebar (which populates the sidebar context), then without closing the sidebar go to an ACQ file (via the PIMS files tab of a property of the file). Then the container renders what is already in context thinking it is an ACQ file when it is not.